nimbus.fml.yaml

```diff
@@ -12,12 +12,12 @@ features:
         type: MR2022Section
         default:
         {
-          "onboarding-first-run-flow": false,
-          "onboarding-upgrade-flow": false,
-          "sync-cfr": false,
-          "jump-back-in-cfr-update": false,
-          "toolbar-cfr-update": false,
-          "wallpaper-onboarding-sheet": false,
+          "onboarding-first-run-flow": true,
```

Comment:
nit: if the default is set to true will it make sense to remove the beta and developer channel values?

Reply:
This is a good point.